astropy/io/ascii/ecsv.py

```diff
@@ -7,16 +7,24 @@
 import re
 from collections import OrderedDict
 import warnings
+import json
+
+import numpy as np
 
 from . import core, basic
 from astropy.table import meta, serialize
 from astropy.utils.data_info import serialize_context_as
-from astropy.utils.exceptions import AstropyWarning
+from astropy.utils.exceptions import AstropyUserWarning, AstropyWarning
+from astropy.io.ascii.core import convert_numpy
 
 __doctest_requires__ = {'Ecsv': ['yaml']}
 
-ECSV_VERSION = '0.9'
+ECSV_VERSION = '1.0'
 DELIMITERS = (' ', ',')
+ECSV_DATATYPES = (
+    'bool', 'int8', 'int16', 'int32', 'int64', 'uint8', 'uint16',
+    'uint32', 'uint64', 'float16', 'float32', 'float64',
+    'float128', 'complex64', 'complex128', 'complex256', 'string')
```

Comment:
I'm confused why `complex` is included here while below it is stated not to be supported.